tox.ini

```diff
@@ -32,37 +38,37 @@ whitelist_externals = /bin/bash
 deps = -rtests_utils/test-requirements.txt
 commands =
     bash -c "set -o pipefail;python setup.py lint | tee pylint.out"
-    bash -c "set -o pipefail;python setup.py pep8 | tee pep8.out"
     bash -c "set -o pipefail;flake8 | tee flake8.out"
 
 
 [testenv:integration]
 whitelist_externals =
     /bin/bash
-    /bin/hostname
-# Use only sitepackages on gateways, no 'deps'
-# Only install 'webtest' as I'm lazy to package it for the moment
-deps = WebTest
-# Requires tox >= 1.8
-sitepackages = True
+    /usr/bin/make
+deps = -rtests_utils/test-requirements.txt
```

Comment:
the WebTest dependency is not used for anything in the end ?

Reply:
It's already installed by the `test-requirements.txt`. The thing is that before it was using the one that is packaged by yocto. I think it was to be as close as possible as a real environment.
When using the test-requirements.txt file, testing dependencies are installed in the tox env. It's simpler this way I think but maybe there are other things I missing.